listeners.py

```diff
@@ -50,7 +56,58 @@ def process_result(self, obj):
                                       obj.result_checksum,
                                       obj.show_reproduce_content))
 
+    def get_failed_task_name(self, task):
+        return task[0][0] + ((':' + task[0][1]) if task[0][1] else '')
+
+    def get_long_mark(self, task):
+        return '(long)' if task in self.long_tasks else ''
+
+    def pretty_task_name(self, task_id):
+        return task_id[0] + ((':' + task_id[1]) if task_id[1] else '')
```

Comment:
Just got that 'pretty' is noun, sorry. It is better to use 'prettify' — verb.

Reply:
Sure, changed.